### PR TITLE
Added support in typing.pyi for PEP 612 (ParamSpec and Concatenate)

### PR DESCRIPTION
**Pull request by erictraut**

### CODE REVIEW

**Comment by gvanrossum** [comment]
@mrkmndz Does this look right to you? (We probably need to add it to typing_extensions.pyi as well.)

File: stdlib/3/typing.pyi [hunk 1]
```diff
@@ -49,6 +49,12 @@ if sys.version_info >= (3, 8):
 if sys.version_info < (3, 7):
     class GenericMeta(type): ...
 
+if sys.version_info >= (3, 10):
+    class ParamSpec:
+        __name__: str
+        def __init__(self, name: str) -> None: ...
+    Concatenate: _SpecialForm = ...
```
Comment (JelleZijlstra) [inline stdlib/3/typing.pyi:56]: The `= ...` is unnecessary (but harmless).

Comment (erictraut) [reply stdlib/3/typing.pyi:56]: It's included everywhere else in this file, so we should be consistent.

**Comment by mrkmndz** [comment]
LGTM!

**Pull request merged**